src/ui/units/collections/components/types.ts

```diff
@@ -3,16 +3,42 @@ import type {CollectionWithPermissions, WorkbookWithPermissions} from 'shared/sc
 
 import {CollectionContentFilters} from '../../../components/CollectionFilters/CollectionFilters';
 
-type CollectionContentProps = {
+export type SelectedMap = Record<
+    string,
+    {
+        type: 'workbook' | 'collection';
+        checked: boolean;
```

Comment:
For what reason do we save unchecked entities in this map?

Reply:
added removing for unchecked